eq_translations/cli/template_extractor.py

```diff
@@ -0,0 +1,29 @@
+import argparse
+import os
+
+from eq_translations.entrypoints import extract_template
+
+def main():
+    parser = argparse.ArgumentParser(description='Extract translation template from json schema')
+
+    parser.add_argument(
+        'SCHEMA_PATH',
+        help='The path to the source schema from which data will be extracted'
+    )
+
+    parser.add_argument(
+        'OUTPUT_DIRECTORY',
+        help='The destination directory for the translation template'
+    )
+
+    args = parser.parse_args()
+
+    if not os.path.isdir(args.OUTPUT_DIRECTORY):
+        print('Output directory does not exist')
+        exit(2)
+
+    extract_template(args.SCHEMA_PATH, args.OUTPUT_DIRECTORY)
```

Comment:
I think we should be more explicit about the potential exit codes for all of the script actions.

Reply:
Looking at this, python will exit with code `1` if an unhandled exception is raised.

Possibly valuable would be to add specific handling for bad file paths etc. But given that there aren't defined exit codes for those issues, I think I will leave this as is for the moment. The traceback and exception will give a fair amount of information to resolve it.